eachlist/Main.java

```diff
@@ -0,0 +1,67 @@
+package eachlist;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        int start, end, threadNumber;
+
+        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
+
+        try {
+            System.out.println("Input start of range: ");
```

Comment:
Refactor. The method is too big. Remove commented lines.

Reply:
done